fuel/datasets/mnist.py

```diff
@@ -1,155 +1,41 @@
 # -*- coding: utf-8 -*-
+import logging
 import os
-import struct
-from collections import OrderedDict
-
-import numpy
 
 from fuel import config
-from fuel.datasets import IndexableDataset
-from fuel.utils import do_not_pickle_attributes
-MNIST_IMAGE_MAGIC = 2051
-MNIST_LABEL_MAGIC = 2049
+from fuel.datasets import H5PYDataset
+
+logger = logging.getLogger(__name__)
```

Comment:
I know a logger is also being created in `binarized_mnist.py` without being used, but could you remove it from this file since it's not being used?

Also, while you're at it, would you mind removing it from `binarized_mnist.py` as well?